package/index.ts

```diff
@@ -317,9 +317,9 @@ export class UploadArea {
     }
   }
 
-  start(autoClear: boolean = false) {
-    if (this.options.manualStart && this.fileList) {
-      this.putFilesToQueue();
+  start(autoClear: boolean = false, files?: IUploadFile[]) {
+    if (this.options.manualStart && (files || this.fileList)) {
+      this.putFilesToQueue(files);
       if (autoClear) this.clear();
```

Comment:
`clear` function has to take files as an argument too.

Reply:
Added.